# app/exceptions/__init__.py
"""Custom domain exceptions"""